mne/viz/utils.py

```diff
@@ -107,6 +107,27 @@ def _check_delayed_ssp(container):
         raise RuntimeError('No projs found in evoked.')
 
 
+def _validate_if_list_of_axes(axes, obligatory_len=None):
+    """ Helper function that validates whether input is a list of axes"""
+    import matplotlib as mpl
+    if obligatory_len is not None and not isinstance(obligatory_len, int):
+        raise ValueError('obligatory_len must be None or int, got %d',
+                         'instead' % type(obligatory_len))
+    if not isinstance(axes, list):
+        raise ValueError('axes must be a list of matplotlib axes objects'
+                         ', got %s instead.' % type(axes))
+    is_correct_type = np.array([isinstance(x, mpl.axes._axes.Axes)
+                               for x in axes])
+    if not np.all(is_correct_type):
+        first_bad = np.where(is_correct_type == False)[0][0]
```

Comment:
flake doesn't like this :( 
it doesn't notice it is numpy array and tells me:
*comparison to False should be `if cond is False:` or `if not cond:`*
Pretty weird, but I'll change to `numpy.logical_not` and remove the comparison